Tests/Shared/StressTestCase.swift

```diff
@@ -4,13 +4,42 @@
 
 import XCTest
 
+/// Base class for stress tests
+///
+/// - runs 100 times if `TestRunnerEnvironment.isStressTest`
+/// - removes all files in `NSTemporaryDirectory()` and `Documents` directory when test suite completes
+/// - by default ends test when test failure occurs
 class StressTestCase: XCTestCase {
     override func setUp() {
         super.setUp()
         
         continueAfterFailure = false
     }
     
+    override class func tearDown() {
+        super.tearDown()
```

Comment:
nit: `super.tearDown` should be called at the end of the method. However, it's probably OK for this particular scenario.